WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift

```diff
@@ -214,9 +196,12 @@ class PlanDetailViewController: UIViewController {
                 }
                 
                 return ImmuTable(sections: groups.map { group in
-                    let features = group.slugs.map { PlanService.featureForPlan(plan, withSlug: $0) }
+                    let rows: [ImmuTableRow] = group.slugs.map { slug in
+                        let feature = PlanService<StoreKitStore>.featureForPlan(plan, withSlug: slug)
+                        return TextRow(title: feature!.title, value: feature!.description)
```

Comment:
You'll probably want to merge in `develop` so you have the latest design changes here.